commands.yaml

```diff
@@ -42,220 +45,190 @@ azmgmt flow export:
-  - '-c, --certificateFile [certificateFile]'
-  - '--thumbprint [thumbprint]'
-azmgmt connect:
-  - '-t, --authType [authType]'
-  - '-u, --userName [userName]'
-  - '-p, --password [password]'
-  - '-c, --certificateFile [certificateFile]'
-  - '--thumbprint [thumbprint]'
-azmgmt logout:
-azmgmt disconnect:
-azmgmt status:
-graph groupsetting add:
+group setting add:
   - '-i, --templateId <templateId>'
-graph groupsetting get:
+group setting get:
   - '-i, --id <id>'
-graph groupsetting list:
-graph groupsetting remove:
+group setting list:
+group setting remove:
   - '-i, --id <id>'
   - '--confirm'
-graph groupsetting set:
+group setting set:
   - '-i, --id <id>'
-graph groupsettingtemplate get:
+group template get:
   - '-i, --id [id]'
   - '-n, --displayName [displayName]'
-graph groupsettingtemplate list:
-graph login:
-  - '-t, --authType [authType]'
-  - '-u, --userName [userName]'
-  - '-p, --password [password]'
-  - '-c, --certificateFile [certificateFile]'
-  - '--thumbprint [thumbprint]'
-graph connect:
-  - '-t, --authType [authType]'
-  - '-u, --userName [userName]'
-  - '-p, --password [password]'
-  - '-c, --certificateFile [certificateFile]'
-  - '--thumbprint [thumbprint]'
-graph logout:
-graph disconnect:
-graph o365group add:
+group template list:
+group add:
   - '-n, --displayName <displayName>'
   - '-d, --description <description>'
   - '-m, --mailNickname <mailNickname>'
   - '--owners [owners]'
   - '--members [members]'
   - '--isPrivate [isPrivate]'
   - '-l, --logoPath [logoPath]'
-graph o365group get:
+group get:
   - '-i, --id <id>'
   - '--includeSiteUrl'
-graph o365group list:
-  - '-d, --displayName [displayName]'
-  - '-m, --mailNickname [displayName]'
+group list:
+  - '-n, --displayName [displayName]'
+  - '-m, --mailNickname [mailNickname]'
   - '--includeSiteUrl'
   - '--deleted'
-graph o365group remove:
+group remove:
   - '-i, --id <id>'
   - '--confirm'
-graph o365group restore:
+group restore:
   - '-i, --id <id>'
-graph o365group set:
+group set:
   - '-i, --id <id>'
   - '-n, --displayName [displayName]'
   - '-d, --description [description]'
   - '--owners [owners]'
   - '--members [members]'
   - '--isPrivate [isPrivate]'
   - '-l, --logoPath [logoPath]'
-graph o365group user add:
-  - '-n, --userName <userName>'
-  - '-i, --groupId [groupId]'
-  - '--teamId [teamId]'
+group user add:
+  - '-u, --userName <userName>'
+  - '--groupId [groupId]'
+  - '-i, --teamId [teamId]'
   - '-r, --role [role]'
-graph teams user add:
-  - '-n, --userName <userName>'
-  - '-i, --groupId [groupId]'
-  - '--teamId [teamId]'
+teams user add:
+  - '-u, --userName <userName>'
+  - '--groupId [groupId]'
+  - '-i, --teamId [teamId]'
   - '-r, --role [role]'
 graph schemaextension add:
   - '-i, --id <id>'
   - '-d, --description [description]'
   - '--owner <owner>'
   - '-t, --targetTypes <targetTypes>'
   - '-p, --properties <properties>'
-graph siteclassification disable:
+site classification disable:
```

Comment:
Site is not a service, so the prefix should be `aad`, eg. `aad siteclassification disable`